packages/functional-tests/pages/resetPassword.ts

```diff
@@ -71,12 +65,16 @@ export class ResetPasswordPage extends BaseLayout {
   }
 
   async resendSuccessMessage() {
-    await this.page.locator(selectors.RESEND_SUCCESS).waitFor();
+    await this.page
+      .locator(selectors.RESEND_SUCCESS)
+      .waitFor({ state: 'visible' });
```

Comment:
Looks like the default for `waitFor` is the visble state.

Reply:
Oh, interesting! I'd seen this used in other tests, but I've removed the option and tests passed. 👍